requirements.pinned.txt

```diff
@@ -51,28 +51,30 @@ flask-alembic==2.0.1      # via invenio-db
 flask-assets==0.12        # via invenio-assets
 flask-babelex==0.9.3
 flask-breadcrumbs==0.4.0  # via invenio-accounts, invenio-oauth2server, invenio-oauthclient, invenio-theme, invenio-userprofiles
-flask-caching==1.3.3      # via invenio-cache
-flask-celeryext==0.3.0    # via invenio-celery, invenio-files-rest, invenio-indexer, invenio-opendefinition, invenio-records
+flask-caching==1.4.0      # via invenio-cache
+flask-celeryext==0.3.1    # via invenio-accounts, invenio-celery, invenio-files-rest, invenio-indexer, invenio-opendefinition, invenio-records
 flask-collect==1.2.2      # via invenio-assets
-flask-cors==3.0.3         # via invenio-rest
+flask-cors==3.0.4         # via invenio-rest
 flask-debugtoolbar==0.10.1
-flask-iiif==0.3.2
+flask-iiif==0.4.0
 flask-kvsession==0.6.2    # via invenio-accounts
 flask-login==0.4.1        # via flask-security, invenio-accounts, invenio-admin, invenio-files-rest, invenio-oauth2server
-flask-mail==0.9.1         # via flask-security, invenio-mail, invenio-userprofiles
+flask-mail==0.9.1         # via flask-security, invenio-accounts, invenio-mail, invenio-userprofiles
 flask-menu==0.7.0         # via flask-breadcrumbs, invenio-accounts, invenio-admin, invenio-theme, invenio-userprofiles
 flask-oauthlib==0.9.4     # via invenio-oauth2server, invenio-oauthclient
 flask-principal==0.4.0    # via flask-security, invenio-admin
 flask-restful==0.3.6      # via flask-iiif
 flask-security==3.0.0     # via invenio-accounts
 flask-sqlalchemy==2.3.2   # via flask-alembic, invenio-db
 flask-wtf==0.14.2
-flask==0.12.2
+flask==1.0.2
```

Comment:
I think for this you need to upgrade `invenio-webhooks`, no?
https://github.com/inveniosoftware/invenio-webhooks/commit/f407cb2245464543ee474a81189fb9d3978bdde5

Reply:
I rebased the PR already 😉